guide/daily-spending-wallet/privacy.md

```diff
@@ -45,222 +46,101 @@ This page should inform about what information is made public when sending or re
 
 </div>
 
----
-
-### Design with privacy in mind
-
-Thinking about privacy is critical during the design process. Your users will not have the same level of knowledge of how to use bitcoin privately. The [privacy by design framework](https://www.ipc.on.ca/wp-content/uploads/Resources/7foundationalprinciples.pdf) states that privacy should be incorporated and built into products by default. This way, whether or not the user is concerned with their data privacy, they would always be protected through good UX and UI.
+## Other privacy aspects
 
-However, it still is important to help users understand any actions that might impact their privacy. Most of the risks occur at the point of creating a transaction or requesting a payment, and we should try to design solutions that reduce the risk of unknowingly degrading privacy.
-
-While there is no perfect solution that will guarantee 100% privacy, try to minimize how much information gets shared to the most essential. Consider ways to inform and prevent user actions that negatively impact their privacy as they use your product.
-
-<!--
-There is no perfect solution to guarantee 100% privacy that lasts forever because things can be revealed over time. Since transactions are forever public, even if all precautions are taken at the time of payment to ensure the highest degree of anonymity, future behaviors of the wallet owner or transacting parties can still degrade previously attained privacy. A high amount of diligence is necessary whenever users are transacting with bitcoin. The product should be able to guide, inform, and prevent them against privacy degrading actions.
--->
+On-chain [address re-use]({{ "/guide/how-it-works/wallet-privacy/#generate-a-new-address-for-each-on-chain-payment" | relative_url }}) is a common privacy problem due to the tracability across transactions. Addresses this wallet uses are swap addresses generated by a lightning service provider. This lifts the responsibility for address management out of the hands of the user. More on the [requesting page]({{ "/guide/daily-spending-wallet/requesting/" | relative_url }}).
 
-[^1]: [Bitcoin.org - Protecting your privacy](https://bitcoin.org/en/protect-your-privacy)
-[^2]: [Top Seven Ways Your Identity Can Be Linked to Your Bitcoin Address](https://99bitcoins.com/know-more-top-seven-ways-your-identity-can-be-linked-to-your-bitcoin-address/)
-[^3]: [Bitcoin Wiki / Privacy](https://en.bitcoin.it/wiki/Privacy#Common-input-ownership_heuristic)
+Just like any other application, privacy can be impacted by the use of third-party services (analytics, currency exchange rates, etc) and data transfer itself, as well as any services provided by the application developer. For more on this, view the [wallet privacy]({{ "/guide/how-it-works/wallet-privacy/" | relative_url }}) page.
 
```

Comment:
This page needs something on sharing invoices as they contain a lot of sensitive info. Maybe just something about how invoices should be shared over private communication channels. 

Using LSPs / LWS have some privacy considerations also, may want to add content around this

Reply:
What are specific examples that can be included on this page?